index.html

```diff
@@ -1992,6 +1999,9 @@ <h3>Processing capabilities</h3>
 
    <dt>"<code>setWindowRect</code>"
    <dd>Boolean indicating whether the <a>remote end</a> supports all of the <a href=#resizing-and-positioning-windows>resizing and positioning</a> <a>commands</a>.
+
+   <dt>"<code>userAgent</code>"
+   <dd>String containing the <a>default User-Agent value</a>.
```

Comment:
To be in sync with the other string capabilities:

```suggestion
   <dd>The <a>default user agent value</a> as a <a>string</a>.
```

Also for both the the `browserName` and `browserVersion` we use `user agent`. Should we use that here as well and update the dfn below?

Reply:
"default User-Agent value" is a link referencing another spec. I think the user agent in other capabilities is the infra spec user agent.

Reply:
Would linking still work if the link name is not an exact match?

Reply:
Looks like we can resolve this issue.